src/main/java/edu/kis/powp/jobs2d/drivers/DriverChangeObserver.java

```diff
@@ -0,0 +1,11 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import edu.kis.powp.jobs2d.features.DriverFeature;
+import edu.kis.powp.observer.Subscriber;
+
+public class DriverChangeObserver implements Subscriber {
```

Comment:
The name should be more about, what it does (not only what it observers).

Reply:
Will DriverChangeTitleObserver be better name?